### PR TITLE
Data protection for user profile data

### PR DESCRIPTION
**Pull request by joinimran**
As the user has reported that if user profile data is in any other drive instead of Windows installed profile location WDAG give an error. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4377

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/enable-controlled-folders-exploit-guard.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ For more information about disabling local list merging, see [Prevent or allow u
 >If controlled folder access is configured with Group Policy, PowerShell, or MDM CSPs, the state will change in the Windows Security app after a restart of the device.
 >If the feature is set to **Audit mode** with any of those tools, the Windows Security app will show the state as **Off**.
 
+>If you are protecting user profile data, it is recommended that user profile should be on default Windows installation drive.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/enable-controlled-folders-exploit-guard.md]:
```suggestion
>If you are protecting user profile data, it is recommended that the user profile should be on the default Windows installation drive.
```

**Comment by joinimran** [comment]
Suggestions applied, ready to go for next level.

File: windows/security/threat-protection/windows-defender-exploit-guard/enable-controlled-folders-exploit-guard.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ For more information about disabling local list merging, see [Prevent or allow u
 >If controlled folder access is configured with Group Policy, PowerShell, or MDM CSPs, the state will change in the Windows Security app after a restart of the device.
 >If the feature is set to **Audit mode** with any of those tools, the Windows Security app will show the state as **Off**.
 
+>If you are protecting user profile data, it is recommended that the user profile should be on the default Windows installation drive.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/enable-controlled-folders-exploit-guard.md]:
```suggestion
>If you are protecting user profile data, we recommend that the user profile should be on the default Windows installation drive.
```

**Comment by joinimran** [comment]
@mypil Ready to go.

**Comment by mypil** [comment]
@levinec - PR has been copy edited and is ready for final review. This is based on issue #4377.

Thank you.

**Pull request merged**